frontend/src/forms/__snapshots__/ParcelDetailForm.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ParcelDetailForm renders correctly 1`] = `
+exports[`ParcelDetailForm ParcelDetailForm renders correctly 1`] = `
```

Comment:
Any reason for the duplicate "ParcelDetailForm"?

Reply:
This is auto-generated by React so don't know what changed here...